ui/modules/things/things.html

```diff
@@ -92,24 +92,26 @@ <h5 data-bs-toggle="collapse" data-bs-target="#collapseAttributes">Attributes <s
 <hr />
 <div class="collapse" id="collapseAttributes">
     <div class="row">
```

Comment:
Please add class `resizable_pane` and `style="height: 20vh;"` so that a minimum height is given (if there is no or only one or two attributes). The height of the ace editor is then resizable.